src/Impostor.Server/Net/State/Game.Incoming.cs

```diff
@@ -43,6 +44,12 @@ public async ValueTask HandleEndGame(IMessageReader message, GameOverReason game
                 player.Value.Limbo = LimboStates.PreSpawn;
             }
 
+            // Delete all PlayerInfo objects
+            foreach (var playerInfo in GameNet.GameData.Players.Values.ToList())
```

Comment:
Actually it seems that `ToList` is more performant and allocates less memory: https://stackoverflow.com/a/60103725

I'm not concerned with scraping the bottom of the performance barrel, it'd be preferable to me to get rid of this copy action completely, that should improve performance in all cases.

Reply:
It does not, DespawnPlayerInfoAsync only despawns server owned playerinfo's.

Also please stop hijacking review conversations with a different concern

Reply:
Okay you win, according to the benchmarks there is an optimization in case the array size is long. I haven't benchmarked it but I just want to get this all over with at this point